modules/arrow/src/parsers/parse-arrow-in-batches.ts

```diff
@@ -37,6 +37,7 @@ export function parseArrowInBatches(
           shape: 'arrow-table',
           batchType: 'data',
           data: new arrow.Table([recordBatch]),
+          rawArrayBuffer: asyncIterator,
```

Comment:
This  (including the input async iterator in the batch) raises red flags. What are you trying to achieve?

Reply:
This should be removed.